ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -642,22 +642,9 @@ func (r *RayClusterReconciler) reconcilePods(ctx context.Context, instance *rayv
 	// Reconcile worker pods now
 	for _, worker := range instance.Spec.WorkerGroupSpecs {
 		// workerReplicas will store the target number of pods for this worker group.
-		var workerReplicas int32
-		// Always honor MaxReplicas if it is set:
-		// If MaxReplicas is set and Replicas > MaxReplicas, use MaxReplicas as the
-		// effective target replica count and log the discrepancy.
-		// See https://github.com/ray-project/kuberay/issues/560.
```

Comment:
Is this comment no longer relevant?

Reply:
Move the comments to `GetWorkerGroupDesiredReplicas`.